### PR TITLE
Fixed hero section on the scholarX page

### PR DESCRIPTION
**Pull request by BhathiyaTK**
## Purpose
Solve #440 

## Goals
Fix the hero section on the ScholarX page.

## Approach
Added `img/scholarx/findmentor.png` image to the hero section as the given design.

### Screenshots
Screenshot (75)

### Preview Link
https://pr-446-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
N/A 

## Test environment
- Firefox Developer Edition v73.0b6, 
- Google Chrome Dev v81.0.4029.3, 
- Opera v66.0.3515.36, 
- Microsoft Edge v44.18362.449.0

## Learning
N/A

### CODE REVIEW

**Comment by jayasanka-sack** [comment]
@BhathiyaTK Replace the surge link

**Comment by BhathiyaTK** [comment]
Surge link replaced.

**Review by jayasanka-sack — DISMISSED** [review]

File: src/main/webapp/scholarx.html [hunk 1]
```diff
@@ -31,11 +31,11 @@
 <section class="section pb-0 bg-gradient-white">
     <div class="container">
         <div class="row row-grid align-items-center justify-content-center">
-            <div class="col-md-8 order-lg-1 text-center">
+            <div class="col-md-6">
                 <div class="d-flex px-3">
                     <div class="pl-4">
                         <img src="assets/img/brand/scholarx.png" class="img-fluid align-self-center mb-5"
-                             style="width: 40%">
+                             style="width: 50%">
```
Comment (Gravewalker666) [inline src/main/webapp/scholarx.html]: @BhathiyaTK 
It's a great great job you have done with the output. However, as a best practice we try to use less inline CSS codes.
Thus it'll be awesome and more readable if we can use bootstrap classes.
Try using w-50 class here.
- Read this documentation for a better understanding https://getbootstrap.com/docs/4.4/utilities/sizing/#relative-to-the-parent

Cheers Mate!
For betterment!!

File: src/main/webapp/scholarx.html [hunk 2]
```diff
@@ -45,6 +45,10 @@
                     </div>
                 </div>
             </div>
+            <div class="col-md-6">
+                <img src="assets/img/scholarx/findmentor.png" class="img-fluid align-self-center"
+                             style="width: 100%">
```
Comment (Gravewalker666) [inline src/main/webapp/scholarx.html]: w-100 class will do the same as this.

File: src/main/webapp/scholarx.html (hunk 1 above)
Comment (BhathiyaTK) [reply src/main/webapp/scholarx.html]: 😍 Sure. I will update.

File: src/main/webapp/scholarx.html (hunk 2 above)
Comment (BhathiyaTK) [reply src/main/webapp/scholarx.html]: Okay. I will update it too.

**Review by jayasanka-sack — APPROVED** [review]

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Add @Gravewalker666 's suggestion

**Comment by BhathiyaTK** [comment]
@jayasanka-sack added @Gravewalker666 's suggest.

**Review by jayasanka-sack — APPROVED** [review]

**Review by Gravewalker666 — APPROVED** [review]

**Pull request merged**